webapp/TargetedMS/js/QCTrendPlotPanel.js

```diff
@@ -417,6 +417,15 @@ Ext4.define('LABKEY.targetedms.QCTrendPlotPanel', {
                 padding: '0 10px 10px 10px',
                 items: toolbarItems
             });
+
+            //hiding the show All series in a single plot checkbox for run scoped metrics
+            var items = this.otherPlotOptionsToolbar.items.items;
```

Comment:
Do we need to reshow later? Or does this get completely reinitialized so there's no need?

Reply:
Yes, we need to reshow this when molecule scoped metric is selected from the dropdown. It does not get reinitialized.

Reply:
Are all of the shows and hides operating on the same components? It seemed odd to me that we have two sets of hide() calls and only one set of show(). And that they span indices -1 to +2, and -2 to +1.

Reply:
Yes, they are operating on same components and the range includes the separators and tab spacers associated with the checkbox.